client/app/pages/users/UsersList.jsx

```diff
@@ -256,6 +274,11 @@ export default function init(ngModule) {
       title: 'Users',
       key: 'active',
     },
+    {
+      path: '/users/pending',
+      title: 'Invited Users',
```

Comment:
Not sure what the standard naming is for this. Perhaps `Pending Invites`?

Reply:
Yes, it's a subject to discuss

Reply:
There is the only user that is not invited :smile: